scripts/jtsync/README.md

```diff
@@ -0,0 +1,57 @@
+# jtsync - synchronize status from jenkins to trello
+    Usage: jtsync --ci SERVICE (--matrix|--job) JOB_STATUS
+          --ci SERVICE                 Which ci is used (suse or opensuse)
+          --matrix NAME,PROJECT        Set status of a matrix job
+          --job NAME                   Set status of a normal job
+
+Matrix job cards are selected in this form:
+
+    $ jtsync --ci suse --matrix crowbar-trackupstream,Devel:Cloud:7:Staging
+
+The script never returns anything other than `0` to make sure jenkins run are not interruped
+by the script.
+
+
+## Run the script on local machine
```

Comment:
Sure but required to run the test: I rephrase to "Run the script on a developer machine"?